resources/views/admin/view-deleted-events.blade.php

```diff
@@ -0,0 +1,43 @@
+@extends('layouts.app')
+
+@section('content')
+
+
+
+    <table id="table_id" class="display">
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+            <th>Hosted by</th>
+            <th>Timing</th>
+            <th>Location</th>
+            <th>Required Contact info</th>
+            <th>Needed Membership</th>
+            <th>Rating</th>
+            <th>Restore</th>
+        </tr>
+        </thead>
+        <tbody>
+        @foreach ($events as $event)
+            <tr>
+                <td>{{$event->name}}</td>
+                <td>{{$event->description}}</td>
+                <td>{{\App\Organization::findorfail($event->organization_id)->name}}</td>
+                <td>{{$event->timing}}</td>
+                <td>{{$event->location}}</td>
+                @if($event->required_contact_info)  <td>YES</td>    @else <td>NO</td>   @endif
+                @if($event->needed_membership)  <td>YES</td>    @else <td>NO</td>   @endif
+                <td>{{$event->rating}}</td>
+                <td>
+                    <form method="POST" action="{{url('restore_event/'.$event->id)}}">
```

Comment:
Simply, it can be GET function as no data is sent in the request body. Don't forget to edit it in the routes.